cinder/backup/manager.py

```diff
@@ -403,7 +402,9 @@ def create_backup(self, context, backup):
 
             backup.service = self.driver_name
             backup.save()
-            updates = self._run_backup(context, backup, volume)
+
+            # Start backup, then continue_backup, then finish_backup
```

Comment:
This comment is confusing. Can we rephrase it like ... `The backup is done in 3 phases. This starts the first one.`?

Reply:
Done